diff --git a/src/js/level/index.js b/src/js/level/index.js
index d56f027b..5203db86 100644
--- a/src/js/level/index.js
+++ b/src/js/level/index.js
@@ -485,6 +485,17 @@ var Level = Sandbox.extend({

       LevelActions.setLevelSolved(this.level.id, isBest);
       log.levelSolved(this.getEnglishName());
+
+      if (API != null)
+      {
+        var numLevels = 0;
+        LevelStore.getSequences().forEach((s) => {
+          LevelStore.getLevelsInSequence(s).forEach((l) => {
+            numLevels++;
+          });
+        });
+        ScormSaveAnswer(this.level.id, numLevels);
+      }
     }
     this.hideGoal();

diff --git a/src/js/stores/LevelStore.js b/src/js/stores/LevelStore.js
index 795e51a1..a00f8462 100644
--- a/src/js/stores/LevelStore.js
+++ b/src/js/stores/LevelStore.js
@@ -186,17 +186,19 @@ AppConstants.StoreSubscribePrototype,
   },

   isLevelSolved: function(levelID) {
-    var levelData = _solvedMap[levelID];
-    if (levelData === true) {
-      return true;
+    var sd = ScormProcessGetValue("cmi.suspend_data");
+    if (sd != null)
+    {
+      var sd_arr = sd.split(",");
+      return sd_arr.includes(levelID);
     }
-    return levelData ? levelData.solved === true : false;
+
+    return false;
   },


   isLevelBest: function(levelID) {
-    var levelData = _solvedMap[levelID];
-    return levelData ? levelData.best === true : false;
+    return false;
   },


diff --git a/src/style/font-awesome.css b/src/style/font-awesome.css
index 57f6f1fe..b9f8a03c 100644
--- a/src/style/font-awesome.css
+++ b/src/style/font-awesome.css
@@ -23,8 +23,8 @@
     */
 @font-face {
   font-family: "FontAwesome";
-  src: url('../assets/font/fontawesome-webfont.eot');
-  src: url('../assets/font/fontawesome-webfont.eot?#iefix') format('eot'), url('../assets/font/fontawesome-webfont.woff') format('woff'), url('../assets/font/fontawesome-webfont.ttf') format('truetype'), url('../assets/font/fontawesome-webfont.svg#FontAwesome') format('svg');
+  src: url('assets/font/fontawesome-webfont.eot');
+  src: url('assets/font/fontawesome-webfont.eot?#iefix') format('eot'), url('assets/font/fontawesome-webfont.woff') format('woff'), url('assets/font/fontawesome-webfont.ttf') format('truetype'), url('assets/font/fontawesome-webfont.svg#FontAwesome') format('svg');
   font-weight: normal;
   font-style: normal;
 }
diff --git a/src/template.index.html b/src/template.index.html
index b04c1d56..d6c9197a 100644
--- a/src/template.index.html
+++ b/src/template.index.html
@@ -9,10 +9,9 @@
 
   <meta property="og:title" content="Learn Git Branching"/>
   <meta property="og:type" content="website"/>
-  <meta property="og:url" content="https://pcottle.github.io/learnGitBranching/index.html?demo"/>
-  <meta property="og:image" content="https://pcottle.github.io/learnGitBranching/assets/learnGitBranching.png"/>
   <meta property="og:description" content="An interactive Git visualization tool to educate and challenge!"/>
   <meta http-equiv="content-language" content="en">
+  <script src="js/scorm.js" type="text/javascript"></script>
 
   <link rel="stylesheet" href="{{buildDir}}/{{styleFile}}" type="text/css" charset="utf-8">
   <link rel="icon" href="assets/favicon.ico" type="image/x-icon">
@@ -359,7 +358,7 @@
   -->
   <script defer src="{{buildDir}}/{{jsFile}}"></script>
 </head>
-<body>
+<body onload="ScormProcessInitialize();" onbeforeunload="ScormProcessTerminate();" onunload="ScormProcessTerminate();">
   <!-- lol noscript. I did not write 9k+ lines of JS just for you to bounce off my website -->
   <noscript>
     <div style="z-index:1000000;background:red;color:white;position:absolute;top:0;left:0;height:300px;width:300px;">
@@ -460,18 +459,5 @@
 </style>
 
 
-  <!-- Google tag (gtag.js) -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-SF8FGS1DYX"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
-
-    gtag('config', 'G-SF8FGS1DYX');
-  </script>
 </body>
 </html>
